portal/plugins/gnmatomresponder/master_importer.py

```diff
@@ -174,28 +174,77 @@ def register_pac_xml(self, content):
         record.save()
         return record
 
+    def check_for_old_finished_jobs(self, vs_item_id):
+        from models import ImportJob
+
+        jobs = ImportJob.objects.filter(item_id=vs_item_id).filter(status='FINISHED')
+
+        number = len(jobs)
```

Comment:
you might be able to simplify this to `return len(jobs)>0`? or that might be javascript. try and see if it works....

Reply:
This seems to work fine. I have added it in three places.